colony/commands/sb.py

```diff
@@ -26,31 +26,33 @@ class SandboxesCommand(BaseCommand):
 
     options:
        -h --help                        Show this message
-       -d, --duration <minutes>         Sandbox will automatically deprovision at the end of the provided duration
-       -n, --name <sandbox_name>        Provide name of Sandbox. If not set, the name will be generated using timestamp
+       -d, --duration <minutes>         The Sandbox will automatically de-provision at the end of the provided duration
+       -n, --name <sandbox_name>        Provide a name for the Sandbox. If not set, the name will be generated
+                                        automatically using the source branch (or local changes) and current time.
 
-       -i, --inputs <input_params>      Comma separated list of input parameters. Example: key1=value1, key2=value2.
-                                        By default Colony CLI will try to take default values for inputs from blueprint
-                                        definition yaml file (if you are inside a git-enabled folder of blueprint repo).
-                                        Use this option to override them.
+       -i, --inputs <input_params>      The Blueprints inputs can be provided as a comma-separated list of key=value
+                                        pairs. For example: key1=value1, key2=value2.
+                                        By default Colony CLI will try to take the default values for these inputs
+                                        from the Blueprint definition yaml file.
```

Comment:
Maybe also add something like (perhaps you can suggest better phrasing):
"You can override all or some of the input default values using the '--inputs' flag"

Reply:
Not sure I understand - it already says that no?

Reply:
I guess it does in some way, but not very explicitly. Ok I will drop it